app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -46,6 +46,7 @@ def start
     image = target_entity
     return queue_signal(:abort_job, "no image found", "error") unless image
     return queue_signal(:abort_job, "cannot analyze non docker images", "error") unless image.docker_id
+    # return queue_signal(:abort_job, "role-missing", "error") unless CURRENT_SERVER.active_roles.include?("smartproxy")
```

Comment:
@roliveri @jrafanie I could not seem to find the right way to do the above, care to suggest something?